cmd/user-agent/src/main.js

```diff
@@ -44,7 +44,7 @@ let defaultAriesStartupOpts = {
     assetsPath: '/aries-framework-go/assets',
     'outbound-transport': ['ws', 'http'],
     'transport-return-route': 'all',
-    'http-resolver-url': [],
+    'http-resolver-url': ["trustbloc@http://localhost:9080/1.0/identifiers"],
```

Comment:
This looks like a local environment path.

Reply:
@talwinder50 remove value from here

Reply:
add it here https://github.com/trustbloc/edge-agent/blob/master/test/bdd/fixtures/agent-wasm/docker-compose.yml#L15

Reply:
https://github.com/trustbloc/edge-agent/blob/master/cmd/http-server/startcmd/start.go#L78